src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -324,7 +327,8 @@ class ActivityFeed extends React.Component<Props, State> {
     if (type === TRANSACTION_EVENT) {
       const tag = get(notification, 'tag', '');
       const isReceived = addressesEqual(notification.to, activeAccountAddress)
-        || tag === PAYMENT_NETWORK_ACCOUNT_WITHDRAWAL;
+        || tag === PAYMENT_NETWORK_ACCOUNT_WITHDRAWAL
+        || bitcoinAddresses.some(e => e.address === notification.to);
```

Comment:
`|| bitcoinAddresses.some(e => addressesEqual(e.address, notification.to));`

Reply:
Bitcoin addresses have upper and lower case letters, we should not use case-insensitive matching.

Reply:
Eth addresses have upper and lower case letters as well, sometimes different api could return you the same hash but with differently capitalized letters and that's why we're using addressesEqual() helper

Reply:
Yes, on ethereum addresses are hashes, so they are the same no matter the capitalization. On bitcoin they're not hashes, and capitalization matters. No API should change the capitalization of bitcoin addresses, since they'd be different (or even invalid) addresses.

Reply:
got it! good to know ;)

Reply:
Sorry, maybe "hashes" vs "not hashes" is misleading, I mean they're not hexadecimal numbers or anything like that.

Reply:
:D 👍